gsa/package.json

```diff
@@ -23,6 +23,8 @@
     "extends": "react-app"
   },
   "dependencies": {
+    "@greenbone/components": "git+https://github.com/chaoa/ui-components.git",
+    "@material-ui/core": "4.11.0",
```

Comment:
Do we really need this import here? Should it be a sub-dependency of @greenbone/components